src/services/userService.js

```diff
@@ -20,6 +23,20 @@ class UserService {
       throw error;
     }
   }
+
+  /**
+   * @returns {*} users
+   */
+  static async getAllEngineers() {
+    const users = await database.User.findAll({
+      attributes: {
+        exclude: ['password'],
```

Comment:
Thanks for reviewing the PR, I will remove lines of excluding the password, but if you don't mind can you clarify what parameters and the use of those parameters we need to pass because the function of that method is to get all LF's engineers and it satisfies that. looking forward to hearing from you.

Reply:
What I was saying is that if you check the PR above there is a function that does the same thing

Reply:
Okay got it. I used the above function as you suggested.